grafana-operator/grafana-operator
# esVersion support both int or str

## Description
Use intOrStr on esVersion in the datasource to be compatible with both grafana 7 and 8.
The only thing that have been changed in the GrafanaDataSourceJsonData struct is the esVersion, the rest is just indentation.

Fixes #753

## Relevant issues/tickets
#753 

## Type of change


## Checklist


## Verification steps

```shell
make run
k apply -f deploy/examples/datasources/es.yaml
```
port-forward to the deployment and verify that the datasource is getting the version set.
